x/ccv/child/module.go

```diff
@@ -249,6 +278,32 @@ func (am AppModule) OnChanOpenInit(
 		return err
 	}
 
+	///////////////////////////////////////////////////
+	// Initialize distribution token transfer channel
+	// NOTE the handshake for this channel is handled by
+	//      the ibc-go/transfer module
+
+	transferPortID := "transfer"
+	am.keeper.SetDistributionTransmissionPort(ctx, transferPortID)
+
+	distChan := channeltypes.Channel{
+		State:          channeltypes.UNINITIALIZED,
+		Ordering:       channeltypes.UNORDERED,
+		Counterparty:   counterparty,          // XXX is it correct to reuse
```

Comment:
```suggestion
		Counterparty:   channeltypes.NewCounterparty(transfertypes.Version, ""),
```

Reply:
we're basically asking for a counterparty channel with port transfer. ChannelID is left blank, since it will be autogenerated on counterparty and returned to us later in the handshake

Reply:
Thanks for the clarification. I've included this logic in the newest commit with some more comments reflecting your explanation.